fetch/cross-origin-resource-policy/scheme-restriction.any.js

```diff
@@ -0,0 +1,7 @@
+// META: script=/common/get-host-info.sub.js
+
+promise_test(t => {
+  return promise_rejects(t,
+                         new TypeError(),
+                         fetch(get_host_info().HTTPS_REMOTE_ORIGIN + "/fetch/cross-origin-resource-policy/resources/hello.py?corp=same-site", { mode: "no-cors" }));
+}, "Cross-Origin-Resource-Policy: same-site's scheme restriction");
```

Comment:
Nit: Perhaps name the test "`same-site` will block an HTTPS resource's inclusion into an HTTP document." for clarity?